Codeinwp/optimole-wp
# Add welcome notice

### All Submissions:

 

### Changes proposed in this Pull Request:
 
- Removes unused admin body class `optimole-optin-show`;
- Adds new welcome notice inside the dashboard (see design below);
- Adds 256x256 plugin icon;

![image](https://user-images.githubusercontent.com/15010186/199194838-d507e7b2-0679-4cfb-872a-31f1aa9b4614.png)

Closes Codeinwp/optimole-service#622

### How to test the changes in this Pull Request:

1. Install the PR build on a new instance and activate the plugin;
2. The plugin should redirect to the dashboard after manual activation on the plugins page;
3. IF the redirect doesn't happen (you can simulate that by running `wp option update optml_notice_optin no` in WP CLI), the notice should be visible on the dashboard pages;
5. It will be dismissed when clicking either of the buttons inside it;
 
### Other information:
